src/plugins/discover/public/application/components/default_discover_table/table_row.tsx

```diff
@@ -111,7 +111,9 @@ const TableRowUI = ({
             <td
               key={colName}
               data-test-subj="docTableField"
-              className="osdDocTableCell eui-textBreakAll eui-textBreakWord"
+              className={`osdDocTableCell ${
+                colName === '@timestamp' ? 'eui-textNoWrap' : 'eui-textBreakAll eui-textBreakWord'
```

Comment:
does the column name is fixed as `@timestamp`, i think we can select different filed as time field for index pattern
![image](https://github.com/user-attachments/assets/9187a97c-15ed-4f79-b256-15f23482af1c)

Reply:
I dont quite understand this

Reply:
> I dont quite understand this
> 
As you hardcode the column name as `@timestamp`, do we have other column name?

Reply:
have updated the condition for it add this for only timefield columns